zulip_bots/zulip_bots/bots/tictactoe/tictactoe.py

```diff
@@ -287,11 +293,17 @@ def handle_message(self, message, bot_handler):
         storage = bot_handler.storage
         if not storage.contains(original_sender):
             storage.put(original_sender, None)
+
+        # Some types are ignored later because `storage.get` is annotated to
+        # return a string only. However, it can actually return other types,
+        # like the board. For this reason, MyPy thinks `user_board`'s type is a
+        # `str` when it's actually a `list`, so `type: ignore` is used.
```

Comment:
I've not run into this before; I'll be interested to see if this remains an issue when you're using the capitalized list & dict forms.